api_tests/nodes/views/test_node_draft_registration_list.py

```diff
@@ -90,15 +105,96 @@ def metadata(draft):
             return test_metadata
         return metadata
 
+    @pytest.fixture()
+    def schema(self):
+        return RegistrationSchema.objects.get(
+            name='OSF-Standard Pre-Data Collection Registration',
+            schema_version=SCHEMA_VERSION
+        )
+
+    @pytest.fixture()
+    def metaschema_open_ended(self):
```

Comment:
nit/honest question: why the separate schema?

Reply:
I think the separate `metaschema_open_ended` fixture is there to ensure draft registrations are tested with the right schema for consistency.